Sources/Sentry/include/HybridPublic/SentrySessionReplayIntegration-Hybrid.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+#import <Sentry/SentryDefines.h>
+
+#if SENTRY_UIKIT_AVAILABLE
+@class SentryReplayOptions;
+
+@protocol SentryViewScreenshotProvider;
+@protocol SentryReplayBreadcrumbConverter;
+
+@interface SentrySessionReplayIntegration : NSObject
+
+- (void)startWithOptions:(SentryReplayOptions *)replayOptions
+      screenshotProvider:(id<SentryViewScreenshotProvider>)screenshotProvider
+     breadcrumbConverter:(id<SentryReplayBreadcrumbConverter>)breadcrumbConverter
```

Comment:
I don't know much about objective-c, but shouldn't this be:
```suggestion
     breadcrumbConverter:(id<SentryReplayBreadcrumbConverterProtocol>)breadcrumbConverter
```

Reply:
Or, following the convention with `SentryViewScreenshotProvider` which is already a protocol, maybe rename `SentryReplayBreadcrumbConverter` to `SentryDefaultReplayBreadcrumbConverter` and `SentryReplayBreadcrumbConverterProtocol` to `SentryReplayBreadcrumbConverter`.

Actually, I'd also suggest renaming `SentryViewScreenshotProvider` to `SentryScreenshotProvider`

Reply:
> I don't know much about objective-c, but shouldn't this be

No, there is no protocol suffix is the objc side. 
We can work on the class naming.

Reply:
> No, there is no protocol suffix is the objc side.

what about this then?
https://github.com/getsentry/sentry-cocoa/pull/4027/files#diff-75294e7bc3f1e95b081b2d9a83fc65e10468a8c4089a8c0aadd6decf1245a672R5

Reply:
This is swift
```Swift
@objc(SentryReplayBreadcrumbConverter) // <-- the name used in objc
 protocol SentryReplayBreadcrumbConverterProtocol: NSObjectProtocol {
     func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent]
 }
 ```

Reply:
But lets change this